src/Http/Application.php

```diff
@@ -65,7 +63,7 @@ public function __construct(ContainerInterface $container = null)
     {
         parent::__construct($container);
 
-        $this->router = new Router(new StandardRouteParser(), new StandardDataGenerator());
+        $this->router = new Router();
```

Comment:
If it's not a factory, you should try to avoid creating new instances directly and either use factories for those or a container (ideally both)